tests/test_ornithology.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017 Luke Sneeringer
+# Distributed under the MIT License (http://opensource.org/licenses/MIT)
+
+import duck
+from duck.compat import mock
+from duck.ornithology import ANY
+from duck.ornithology import Instance
+
+
+def test_ornithology():
```

Comment:
Ideally it would be nice to have a test against an _actual_ int and float, e.g.

```python
assert Instance(int) == 42
assert Instance(float) == 3.14159
```

That is the thing that actually matters in the end. :-)